src/core/libmaven/csvreports.cpp

```diff
@@ -60,6 +60,10 @@ CSVReports::CSVReports(string filename,
         _insertGroupReportColumnNamesintoCSVFile(
             filename, _prmReport, _includeSetNamesLine);
     }
+
+    else{
+        _reportStream.open(filename.c_str(), ios::out);
+    }

```

Comment:
So, what's this? I don't understand what we are doing here.

Reply:
This for opening the file when it is for function "write for polly". As we need no headings like for peak file and group file. Therefore, It has been given in else part. 
I will add this as comment in the code

Reply:
Can you remind me, why we are having to change things in the CSVReports class in this PR? We probably discussed it, but its been a long time since and I cannot recall what it was. :|

Reply:
Yeaa, because after we refactored Compound class, there was problem of isotopes while writing to csvReports class. 
For example, 'glucose-6-phosphate' was being changed to 'fructose-6-phosphate'.